lib/tic_tac_toe.rb

```diff
@@ -7,17 +7,18 @@ def initialize(board_size = 3)
         @board = []
         @board_top_and_bottom = []
         @board_middle = []
+        @board_middle_section = []
         @board_size = board_size
     end
 
     def go
```

Comment:
Some of the methods in Board could be made private. Are there some methods which you might not want/need to be accessible outside of Board?